backend/src/yeomanui.ts

```diff
@@ -174,15 +174,34 @@ export class YeomanUI {
         console.log("done running yeomanui! " + message + ` You can find it at ${destinationRoot}`);
         this.doGeneratorDone(true, message, destinationRoot);
       });
-    } catch (err) {
-      console.error(err);
+    } catch (error) {
+      const errorMessage = this.getErrorInfo(error);
+      console.error(errorMessage);
+      this.logMessage(errorMessage);
```

Comment:
in future we can call logMessage with sevirity prop and print to conle in it.
this.logMessage(errorMessage. LOG.ERROR);